servicepulse/containerization/index.md

```diff
@@ -1,31 +1,84 @@
 ---
 title: Running ServicePulse in containers
-reviewed: 2024-02-26
+reviewed: 2024-08-09
+component: ServicePulse
 ---
 
-Docker images for ServicePulse exist on Docker Hub under the [Particular organization](https://hub.docker.com/u/particular) and are available for Windows and Linux.
+ServicePulse can be deployed on containers using the [`particular/servicepulse` image](https://hub.docker.com/r/particular/servicepulse).
 
-## Containers overview
+The ServicePulse container image includes a reverse proxy feature that allows ServicePulse to act as a single ingress/egress point for a system containing ServiceControl containers. This allows existing container hosting infrastructure to layer additional features onto the ServicePulse ingress point, such as SSL or authentication.
```

Comment:
Should this be highlighted further? For instance as a note or with a header?

Reply:
Good point. Moved to a heading below the usage.

This makes it "match" the other container pages better too.